src/views/Login/login.scss

```diff
@@ -0,0 +1,82 @@
+* {
+  font-family: 'Inter', sans-serif;
+}
+
+.login-container {
```

Comment:
deze class doet het zelfde als main en is daarom onnodig, ik had ook nog main aan de router moeten toevoegen maar alsnog